services/vod-to-live/service/src/common/config/config-definitions.ts

```diff
@@ -117,6 +118,13 @@ export const getConfigDefinitions = (
         .get('CHANNEL_PROCESSING_WAIT_TIME_IN_SECONDS')
         .default('600')
         .asIntPositive(),
+
+    /**
+     * Sets the body size limit for the webhook requests, since playlists can be
+     * quite large. Default is 50mb. Examples: https://github.com/visionmedia/bytes.js#bytesparsestringnumber-value-numbernull
+     */
+    webhookBodySizeLimit: () =>
+      env.get('WEBHOOK_BODY_SIZE_LIMIT').default('50mb').asString(),
```

Comment:
How is this guaranteed to work? The "bytes" library that is linked is not a dependency of the "env-var" package (according to the npm website).

Reply:
The `bytes` library is used internally by `json` component of the body-parser library used by the express app. 
It also supports string/numeric values (as seen in the example link), so I did not see a reason to add hard restrictions on value ending with mb, kb, etc..